akka-stream/src/main/scala/akka/stream/javadsl/Graph.scala

```diff
@@ -136,6 +136,40 @@ object Broadcast {
 }
 
 /**
+ * Fan-out the stream to several streams. emitting an incoming upstream element to one downstream consumer according
+ * to the partitioner function applied to the element
+ *
+ * '''Emits when''' all of the outputs stops backpressuring and there is an input element available
+ *
+ * '''Backpressures when''' one of the outputs backpressure
+ *
+ * '''Completes when''' upstream completes
+ *
+ * '''Cancels when'''
+ *   when one of the downstreams cancel
+ */
+object Partition {
+  /**
+   * Create a new `Partition` stage with the specified input type.
+   *
+   * @param outputCount number of output ports
+   * @param partitioner function deciding which output each element will be targeted
+   */
+  def create[T](outputCount: Int, partitioner: function.Function[T, Int]): Graph[UniformFanOutShape[T, T], Unit] =
+    scaladsl.Partition(outputCount, partitioner = (t: T) ⇒ partitioner.apply(t))
```

Comment:
This explicit style is fine (and I’m not saying that it should be changed), but it is equivalent to just saying
~~~scala
scaladsl.Partition(outputCount, partitioner.apply)
~~~